python_src/tests/performance_manager/test_performance_manager.py

```diff
@@ -65,8 +66,11 @@ def set_env_vars() -> None:
     if int(os.environ.get("BOOTSTRAPPED", 0)) == 1:
         logging.warning("allready bootstrapped")
     else:
-        env_file = os.path.join(HERE, "..", "..", ".env")
+        env_file = os.path.join(
+            os.path.dirname(os.path.abspath(__file__)), "..", "..", "..", ".env"
+        )
```

Comment:
we need this when we're running these tests locally on the command line, but _only_ there.

i think there might be a better way.